src/components/Modal.js

```diff
@@ -38,7 +38,8 @@ const Modal = (props) => {
         swipeDirection,
         animationIn,
         animationOut,
-        needsSafeAreaPadding,
+        shouldSumTopSafeAreaPadding,
+        shouldSumBottomSafeAreaPadding,
```

Comment:
`shouldAddSafeAreaPaddingTop` and `shouldAddSafeAreaPaddingBottom` are maybe more descriptive

Reply:
I swear I started writing with `Add`, have no idea what led me to change.
Fixed.